tests/integration/llm/prepare.py

```diff
@@ -625,6 +625,24 @@
         "option.tensor_parallel_degree": 4,
         "option.enable_chunked_prefill": "true",
     },
+    "llama-68m-speculative-medusa": {
+        "option.model_id": "JackFram/llama-68m",
+        "option.task": "text-generation",
+        "option.speculative_model": "abhigoyal/vllm-medusa-llama-68m-random",
+        "option.num_speculative_tokens": 4,
+        "option.use_v2_block_manager": True,
+        "option.tensor_parallel_degree": 1,
+        "option.max_rolling_batch_size": 4,
+    },
+    "llama-68m-speculative-eagle": {
+        "option.model_id": "JackFram/llama-68m",
+        "option.task": "text-generation",
+        "option.speculative_model": "abhigoyal/vllm-eagle-llama-68m-random",
```

Comment:
Non blocking: we should move these to s3 to avoid long term availability flakiness like we have seen with adapter models.